### ISSUE TITLE
Reinitialize entities of the model before initializing again the model

### ISSUE THREAD

**Issue by SergeStinckwich** [cormas/cormas#238]
When you iniitalize two times the model in the CORMAS UI interface, old agent entities of the previous model are not deleted.
We should remove all entities everytime we initialize the model. Linked to issue: #229

**Comment by ElCep** [comment]
I the same issue seems happend when I try to launch `CMReHabModel simuOpenMole` in the playground :-S

**Comment by ElCep** [comment]
A big investigation today leads us to think that there's a big work in progress around the refactoring of the initialization.

**Comment by SergeStinckwich** [comment]
CMRehabModel is an example where relaunching doesn't work.
When relaunch, the model has an issue in this method:
```Smalltalk
CMReHabModel>>setInitialBiomasses

	| initialPlantBiomasses |
	initialPlantBiomasses := #(1 1 2 1 1 2 0 2 3 2 1 3 1 2 1 1 3 1 0 2).
	self theZones
		with: initialPlantBiomasses
		do: [:z :i | z biomass: i]
```

Apparently ``self theZones`` and ``initialPlantBiomasses`` are collections with different sizes.

**Comment by SergeStinckwich** [comment]
The first step is to understand how model, entities are initialized at the beginning.

**Comment by ElCep** [comment]
ping

**Comment by SergeStinckwich** [comment]
We spend some times with @tomooda to understand how it works ... but still very confused. The idea is to document progressively the initialization to be able to refactor later.

**Comment by ElCep** [comment]
curse!  I had forgotten that it was a complicated problem! But that's it, I remember

**Comment by ElCep** [comment]
My bad ! @pierbo find my bug in ReHab ! I forgotten to initialize grid.... It seem to be solved with PR #290
